README.md

```diff
@@ -42,5 +42,5 @@ Example Usage
 ```sh
 # Download latest Kafka distribution(if necessary)
 $ tar zxvf kafka_LATEST_VERSION.tgz
-$ ./kafka_consumer_group_exporter --consumer-group-command-path=./kafka_LATEST_VERSION/bin/kafka-consumer-groups.sh BOOTSTRAP_SERVERS
+$ ./kafka_consumer_group_exporter --consumer-group-command-path=./kafka_LATEST_VERSION/bin/kafka-consumer-groups.sh BOOTSTRAP_SERVER
```

Comment:
Interesting. I didn't dig into this too much, but I assume it's comma separated? 1a0f1df3bb995d1442d551690742ce9de4ce4855 looks good?